misk-hibernate-testing/src/test/kotlin/misk/jdbc/TruncateTablesServiceTest.kt

```diff
@@ -109,12 +110,9 @@ internal class TruncateTablesServiceTest {
       install(MiskTestingServiceModule())
 
       val config = MiskConfig.load<TestConfig>("test_truncatetables_app", Environment.TESTING)
-      install(HibernateModule(
-          TestDatasource::class, config.data_source))
-      install(object : HibernateEntityModule(
-          TestDatasource::class) {
-        override fun configureHibernate() {
-        }
+      install(HibernateModule(TestDatasource::class, config.data_source))
+      install(object : HibernateEntityModule(TestDatasource::class) {
+        override fun configureHibernate() {}
```

Comment:
hmm. . . . I thought our Spotless forced `{}` to be on two lines. (Special cases are bad)

Reply:
I'll fix it. Didn't realize that was against this was against the style guide.